configure.ac

```diff
@@ -199,6 +199,17 @@ if test "x$enable_host_optimization" = "xno" ; then
 	ARCH_ABI=default-linux
 fi
 
+AC_ARG_ENABLE([wfxt-cpu-pause],
```

Comment:
The commit message could be improved:
- The rationale of the use of WFET in odp_cpu_pause() could be explained
- The function is not cpu_pause but odp_cpu_pause.
- Use of WFET instead of ISB is selected through configure flag, not compiler flag
- The title (and often the message body too) are typically in the imperative form in ODP git